errors/index.js

```diff
@@ -9,7 +9,7 @@ var config = require('../config');
 var logger = require('../lib/logger');
 
 /* eslint no-unused-vars: 0*/
-module.exports = function errorHandler(err, req, res, next) {
+module.exports = function errorHandler(err, req, res) {
```

Comment:
nope, nope, nope

Reply:
Read this: https://expressjs.com/en/guide/error-handling.html

Reply:
ahh ok got some questions:

the documentation says to have 4 args but why do you need to have 4 if you're are not calling next? is it just following the convention?

An interesting bit for myself was `Notice that when not calling “next” in an error-handling function, you are responsible for writing (and ending) the response.` I guess this is happening in res.render at the end.

Reply:
4 args are required by Express or it will parse function as normal middleware and the first argument will be the request object, not the error object.

Yes, if you don't call `next`, you need to end terminate the request in another way, like `res.render` or `res.send`